src/App.css

```diff
@@ -14,6 +14,47 @@
   overflow: hidden;
 }
 
+.landing-headline {
+  color: black;
+  margin-top:50px;
+  font-size: 5em;
+  margin-right: 800px;
+  margin-bottom: 20px;
+  background-color: rgba(254,173,185,0.5);
+/*  text-transform: uppercase;*/
+}
+
+.landing-subhead {
+  color: black;
+  font-size: 1.5em;
+  font-weight: 450;
+  margin-right: 950px;
+  margin-top: 10px;
+  margin-bottom: 7px;
+/*  text-transform: uppercase;*/
+}
+
+    button {
```

Comment:
Maybe we could add a little more padding/ margin to space this out a little bit.

Reply:
Sounds good!